src/astro/databases/sqlite.py

```diff
@@ -37,10 +37,38 @@ def default_metadata(self) -> Metadata:
     # ---------------------------------------------------------
     # Table metadata
     # ---------------------------------------------------------
-    def get_table_qualified_name(self, table: Table) -> str:
+    @staticmethod
+    def get_table_qualified_name(table: Table) -> str:
         """
         Return the table qualified name.
 
         :param table: The table we want to retrieve the qualified name for.
         """
         return str(table.name)
+
+    def populate_table_metadata(self, table: Table) -> Table:
+        """
+        Since SQLite does not have a concept of databases or schemas, we just return the table as is,
+        without any modifications.
+
+        :param table:
+        :return:
```

Comment:
```suggestion
missing docstring, if you don't see value in adding a description where it is crystal clear, feel free to remove it.

Example:

```suggestion
        :param table:
        :return:
```
```

Reply:
@kaxil I'm not sure what you mean here

Reply:
@dimberman, it's about removing the following lines, since they are not giving any meaningful information:
```
        :param table:
        :return:
```